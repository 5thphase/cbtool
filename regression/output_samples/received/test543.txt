 status: Sending a termination request for vm_32 (cloud-assigned uuid 8AC4A4D5-85E3-52D7-AD54-91A01E0E9519)....
 status: Sending a termination request for vm_30 (cloud-assigned uuid 640E69EF-7166-5D2C-83C4-D201B832FF13)....
 status: Sending a termination request for vm_31 (cloud-assigned uuid A8753B85-B52C-59CA-B70A-8471495E24BC)....
 status: Waiting for vm_29 (cloud-assigned uuid 1E79C9EC-55DA-5D5A-8E2B-778889914704) to be captured with image name "vm_29_at_1392746756"...
 status: Sending a termination request for vm_29 (cloud-assigned uuid 1E79C9EC-55DA-5D5A-8E2B-778889914704)....
VM object 5DD6CDFC-A786-5EB5-AC0A-51A2112598D9 (named "vm_29") successfully captured on this experiment.
The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |3.37M                         
|Redis Version                                        |2.6.16                        
|Uptime (in seconds)                                  |2218438                       
|Total Connections Received                           |8152                          
|Total Commands Processed                             |2573805                       
|Number of Keys                                       |549                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.6                         
|Storage Size                                         |1642496                       
|Data Size                                            |612480                        
|Index Size                                           |466032                        
|Average Object Size                                  |522.593856655                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |16                            
|AIs                                                  |6                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |16                            
|VMs ARRIVED                                          |43                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |27                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|AI RESERVATIONS                                      |6                             
|AIs ARRIVED                                          |7                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |1                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |169                           

 ---------------- VMC-WIDE COUNTERS ----------------
|845D544A-3F3E-5433-BD75-03F09FBB3BE7 (simzone_c) VM RESERVATIONS|5                             
|F1774E78-5022-5568-80E1-9A3EFC329DB1 (simzone_b) VM RESERVATIONS|5                             
|6B2CF2C9-7ECD-535A-9836-6D30EBA8F83F (simzone_d) VM RESERVATIONS|4                             
|A3B38AF9-68E8-5E7C-8050-66F6D9681E02 (simzone_a) VM RESERVATIONS|2                             

The following ATTACHED AIs are in this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_1      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |230.63.204.17   |1392746651     |none                                    |733DD717-C43F-5544-AB1A-6451BAB6F774
|ai_3      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |124.172.236.131 |1392746668     |none                                    |CEACB201-4868-5707-9503-0649D9F05808
|ai_4      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |221.203.204.69  |1392746708     |none                                    |2FB238B9-C58D-5D2B-95B9-911BCB6F7EA4
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |105.60.216.110  |1392746726     |none                                    |E831B3C0-E478-56E8-8D99-F0014C035425
|ai_6      |hadoop         |1_x_hadoopmaster->1_x_hadoopslave                 |167.84.222.101  |1392746730     |none                                    |42774CCD-6689-50C1-95CC-18684CBC41E5
|ai_7      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |135.133.64.49   |1392746753     |none                                    |4C31591B-14C6-5CD9-AFE5-96984F263D0F

The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_27         |client_daytrader    |nano32      |230.63.204.17   |simhostd1                  |LG                  |ai_1    |none       |D6D5BD93-2DF3-5250-BA9F-E985D7ABC605
|vm_28         |db2                 |platinum64  |73.5.209.15     |simhosta1                  |SUT                 |ai_1    |none       |4609BC68-A038-5AE3-945F-9D3A27C26DB0
|vm_33         |db2                 |platinum64  |66.31.152.229   |simhostb1                  |SUT                 |ai_3    |none       |3E32506B-B9BA-5A31-9A89-2AED9145DAD7
|vm_35         |client_daytrader    |nano32      |124.172.236.131 |simhostd1                  |LG                  |ai_3    |none       |95C7731F-3A5D-5B02-A386-E6431A186898
|vm_34         |was                 |iron32      |49.2.160.253    |simhosta1                  |SUT                 |ai_3    |none       |F7484D64-8185-589D-96D6-79F04EE763ED
|vm_36         |client_daytrader    |nano32      |221.203.204.69  |simhostd1                  |LG                  |ai_4    |none       |F8A77B67-6E72-59CE-9C63-D69205D75908
|vm_37         |was                 |iron32      |253.252.118.166 |simhostb1                  |SUT                 |ai_4    |none       |AAF940FA-9754-561D-A8B1-0BF359602CD3
|vm_38         |db2                 |platinum64  |149.12.197.87   |simhostc1                  |SUT                 |ai_4    |none       |193977C8-0AEC-5591-87DA-5FE36B212001
|vm_39         |netclient           |platinum64  |105.60.216.110  |simhostc1                  |SUT                 |ai_5    |none       |8CD9EA48-17AB-54CB-8873-2E72D61EDC66
|vm_40         |netserver           |iron32      |111.248.74.195  |simhostb1                  |SUT                 |ai_5    |none       |FC920023-1B0F-594A-9001-13DF3CD531C2
|vm_41         |hadoopmaster        |copper32    |167.84.222.101  |simhostc1                  |SUT                 |ai_6    |none       |34C28B3E-D61A-5AA5-924D-130E98478CC4
|vm_42         |hadoopslave         |platinum64  |174.34.15.164   |simhostc1                  |SUT                 |ai_6    |none       |8435A323-6BFA-5121-AAE9-EE96EDABD031
|vm_43         |was                 |iron32      |68.187.69.148   |simhostb1                  |SUT                 |ai_1    |none       |818601CC-C11C-5A51-A3F2-D8F493D349F2
|vm_45         |db2                 |platinum64  |47.1.59.246     |simhostb1                  |SUT                 |ai_7    |none       |BCD81CCB-B4F8-524D-B14F-834C66EADBAD
|vm_44         |client_daytrader    |nano32      |135.133.64.49   |simhostd1                  |LG                  |ai_7    |none       |84E02885-1680-5D65-AEA8-B40A354BA154
|vm_46         |was                 |iron32      |15.137.233.252  |simhostc1                  |SUT                 |ai_7    |none       |81AE6C61-41EF-5022-BB07-2C89482C51A0

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |3.37M                         
|Redis Version                                        |2.6.16                        
|Uptime (in seconds)                                  |2218438                       
|Total Connections Received                           |8152                          
|Total Commands Processed                             |2574079                       
|Number of Keys                                       |549                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.6                         
|Storage Size                                         |1642496                       
|Data Size                                            |612880                        
|Index Size                                           |466032                        
|Average Object Size                                  |522.489343564                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |16                            
|AIs                                                  |6                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |16                            
|VMs ARRIVED                                          |43                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |27                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|AI RESERVATIONS                                      |6                             
|AIs ARRIVED                                          |7                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |1                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |169                           

 ---------------- VMC-WIDE COUNTERS ----------------
|845D544A-3F3E-5433-BD75-03F09FBB3BE7 (simzone_c) VM RESERVATIONS|5                             
|F1774E78-5022-5568-80E1-9A3EFC329DB1 (simzone_b) VM RESERVATIONS|5                             
|6B2CF2C9-7ECD-535A-9836-6D30EBA8F83F (simzone_d) VM RESERVATIONS|4                             
|A3B38AF9-68E8-5E7C-8050-66F6D9681E02 (simzone_a) VM RESERVATIONS|2                             

