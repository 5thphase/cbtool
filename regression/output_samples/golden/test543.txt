 status: Sending a termination request for vm_30 (cloud-assigned uuid C4DEF8BD-C48E-5264-89C3-B9EF23309BB2)....
 status: Sending a termination request for vm_32 (cloud-assigned uuid 5538084B-27B0-5E3B-B05A-51D3299894DA)....
 status: Sending a termination request for vm_31 (cloud-assigned uuid 45659D6C-487E-5740-94D5-912408E1F0A4)....
 status: Waiting for vm_29 (cloud-assigned uuid CC86B9A7-9620-5F92-B90F-11AEFD8219E6) to be captured with image name "vm_29_at_1399996816"...
 status: Sending a termination request for vm_29 (cloud-assigned uuid CC86B9A7-9620-5F92-B90F-11AEFD8219E6)....
VM object 3273A542-20E7-569B-8B9D-AEDCEBE21224 (named "vm_29") successfully captured on this experiment.
The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |3.51M                         
|Redis Version                                        |2.6.16                        
|Uptime (in seconds)                                  |3793346                       
|Total Connections Received                           |81338                         
|Total Commands Processed                             |6086562                       
|Number of Keys                                       |557                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.9                         
|Storage Size                                         |2158592                       
|Data Size                                            |612656                        
|Index Size                                           |457856                        
|Average Object Size                                  |521.409361702                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |16                            
|AIs                                                  |6                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |16                            
|VMs ARRIVED                                          |43                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |27                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|AI RESERVATIONS                                      |6                             
|AIs ARRIVED                                          |7                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |1                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |169                           

 ---------------- VMC-WIDE COUNTERS ----------------
|516A5A15-B093-5B5F-B171-EE9AA82AAEF0 (simzone_c) VM RESERVATIONS|4                             
|8D79A9F1-34B8-5D1A-A903-082245CE052C (simzone_d) VM RESERVATIONS|4                             
|B0B6FF0A-FAE8-5217-A335-C0F1B0BEA0D6 (simzone_b) VM RESERVATIONS|6                             
|D5506DC9-493A-57EE-86F6-0B96CB07D162 (simzone_a) VM RESERVATIONS|2                             

The following ATTACHED AIs are in this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_1      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |155.224.91.209  |1399996708     |none                                    |98B86D5A-7910-5D31-83FC-749B386BDBF0
|ai_3      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |5.63.8.207      |1399996726     |none                                    |EA4DACC9-4C89-55C8-92BA-7BADF79753D8
|ai_4      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |42.106.201.48   |1399996765     |none                                    |D2AD4C36-E24C-5084-B7AA-38D6E40ED9B7
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |230.121.157.104 |1399996784     |none                                    |6D10147F-0BB2-5DEE-9849-1AF44506C501
|ai_6      |hadoop         |1_x_hadoopmaster->1_x_hadoopslave                 |91.239.57.51    |1399996789     |none                                    |381217E9-5946-5761-94AA-76B91E41965E
|ai_7      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |222.89.149.97   |1399996812     |none                                    |3F291E64-8BB0-56D6-A4CC-DEB5C00DCA12

The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_26         |client_daytrader    |micro32     |155.224.91.209  |simhostb2                  |LG                  |ai_1    |none       |BD695E57-E34F-5617-A755-3EDFF0BDAEAC
|vm_28         |db2                 |platinum64  |225.124.169.215 |simhostc1                  |SUT                 |ai_1    |none       |9F6561D1-8EF0-5675-B7AC-146F4B7552A2
|vm_33         |client_daytrader    |micro32     |5.63.8.207      |simhostc0                  |LG                  |ai_3    |none       |CF7D1E24-C7DF-5748-BEB4-42DED33FC38C
|vm_35         |db2                 |platinum64  |129.42.189.39   |simhosta1                  |SUT                 |ai_3    |none       |D7D3812D-6357-5BF6-9E02-0691B88B7827
|vm_34         |was                 |iron32      |125.217.216.224 |simhosta3                  |SUT                 |ai_3    |none       |5B0D99B7-08AB-565F-AC22-D9F49230D624
|vm_36         |client_daytrader    |micro32     |42.106.201.48   |simhostc2                  |LG                  |ai_4    |none       |B9C6F7C8-93DD-539E-AF51-DF7F5374702C
|vm_38         |db2                 |platinum64  |174.69.72.162   |simhostb0                  |SUT                 |ai_4    |none       |C650DDC3-BCF5-5EBA-887E-8CCD387E3B1B
|vm_37         |was                 |iron32      |24.77.4.207     |simhostd4                  |SUT                 |ai_4    |none       |811D2492-EC9C-5BB1-BEB4-C57E50551C30
|vm_39         |netclient           |platinum64  |230.121.157.104 |simhosta0                  |SUT                 |ai_5    |none       |B38CA59D-17FD-5263-BDB3-8F35D4BD2B9D
|vm_40         |netserver           |iron32      |225.1.162.64    |simhosta3                  |SUT                 |ai_5    |none       |E25FF02C-8BB2-52EB-8F99-C43E02DF751C
|vm_41         |hadoopmaster        |copper32    |91.239.57.51    |simhostb1                  |SUT                 |ai_6    |none       |E9DEBF96-6A58-5026-B961-D4A83A55EEC4
|vm_42         |hadoopslave         |platinum64  |254.26.15.180   |simhosta1                  |SUT                 |ai_6    |none       |A9128D32-E4C1-57F1-94A0-1E9B6BDBFB2A
|vm_43         |was                 |iron32      |181.92.47.136   |simhostb4                  |SUT                 |ai_1    |none       |D797A16E-27E5-5694-B8FD-8ED3AB8F2DA8
|vm_44         |client_daytrader    |micro32     |222.89.149.97   |simhostd2                  |LG                  |ai_7    |none       |3E157FE3-7A06-5EF2-8D62-D451951CBB52
|vm_46         |db2                 |platinum64  |114.96.230.146  |simhostb0                  |SUT                 |ai_7    |none       |BB3D5D20-B153-59CC-98C9-99774272A690
|vm_45         |was                 |iron32      |132.122.28.221  |simhostb0                  |SUT                 |ai_7    |none       |1FF03E74-F0C4-538C-A025-9C4519FC7AD3

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |3.51M                         
|Redis Version                                        |2.6.16                        
|Uptime (in seconds)                                  |3793346                       
|Total Connections Received                           |81338                         
|Total Commands Processed                             |6086836                       
|Number of Keys                                       |557                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.9                         
|Storage Size                                         |2158592                       
|Data Size                                            |613056                        
|Index Size                                           |457856                        
|Average Object Size                                  |521.306122449                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |16                            
|AIs                                                  |6                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |16                            
|VMs ARRIVED                                          |43                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |27                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|AI RESERVATIONS                                      |6                             
|AIs ARRIVED                                          |7                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |1                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |169                           

 ---------------- VMC-WIDE COUNTERS ----------------
|516A5A15-B093-5B5F-B171-EE9AA82AAEF0 (simzone_c) VM RESERVATIONS|4                             
|8D79A9F1-34B8-5D1A-A903-082245CE052C (simzone_d) VM RESERVATIONS|4                             
|B0B6FF0A-FAE8-5217-A335-C0F1B0BEA0D6 (simzone_b) VM RESERVATIONS|6                             
|D5506DC9-493A-57EE-86F6-0B96CB07D162 (simzone_a) VM RESERVATIONS|2                             

